src/api/controllers/members.ts

```diff
@@ -80,6 +80,11 @@ export async function handleInternalGuildMemberUpdate(data: DiscordPayload) {
       guildMember?.nick,
     );
   }
+
+  if (guildMember?.pending === false && cachedMember?.pending) {
```

Comment:
There only guildMember should be checked.

Reply:
It is always present, if there is only check for guildMember?.pending, it'll fire on even nickname changes.

Reply:
That check should be improved then since `cachedMember` should not have the property `.pending`

Reply:
Due to memory efficiency members are cached independent to guilds in DD. So if the .pending parameter would exist in `cachedMemeber` that would mean that the member is pending on all guilds which is not good.